packages/models/test/fixtures/eslint-plugin.mock.ts

```diff
@@ -1,14 +1,44 @@
-import type { CategoryConfig, PluginReport } from '../../src';
+import type {
+  Audit,
+  CategoryConfig,
+  PluginConfig,
+  PluginReport,
+} from '../../src';
 import { ESLINT_AUDITS_MAP } from './eslint-audits.mock';
+import { runnerConfig } from './runner.mock';
+
+const eslintMeta: Omit<PluginConfig, 'audits' | 'runner'> = {
+  slug: 'eslint',
+  title: 'ESLint',
+  icon: 'eslint',
+  description: 'Official Code PushUp ESLint plugin',
+  packageName: '@code-pushup/eslint-plugin',
+  version: '0.1.0',
+};
```

Comment:
This should be easier to manage if the `PluginConfig` changes:

```suggestion
const eslintMeta = {
  slug: 'eslint',
  title: 'ESLint',
  icon: 'eslint',
  description: 'Official Code PushUp ESLint plugin',
  packageName: '@code-pushup/eslint-plugin',
  version: '0.1.0',
} satisfies Partial<PluginConfig>;
```